### ISSUE TITLE
[Disabled for Element-R] Flaky cypress test `Read receipts > Paging up > After marking room as read, paging up to find old threads that were never read leaves the room read`

### ISSUE THREAD

**Issue by andybalaam** [vector-im/element-web#26341]
Flaky test Read receipts > Paging up "After marking room as read, paging up to find old threads that were never read leaves the room read"

**Comment by richvdh** [comment]
https://github.com/matrix-org/matrix-js-sdk/actions/runs/6495998898/job/17642067954#step:8:1374

**Comment by andybalaam** [comment]
> https://github.com/matrix-org/matrix-js-sdk/actions/runs/6495998898/job/17642067954#step:8:1374

This looks like slow startup of the client. Hopefully fixed by https://github.com/matrix-org/matrix-react-sdk/pull/11742

**Comment by richvdh** [comment]
https://github.com/matrix-org/matrix-js-sdk/actions/runs/6590996937/job/17908708726

Looking at the screenshots, there are two different failure modes here.

1. In one, it gets stuck at a loading spinner:

   ![image](https://github.com/vector-im/element-web/assets/1389908/22d4944b-a0d2-4a34-acb0-a22fe6f343d1)

  The logs show it hanging at `Init OlmMachine`. This is ... odd. I don't know why that call would get stuck, nor do I know why only this test would get stuck there.

2. The other failure mode is different:

   ![image](https://github.com/vector-im/element-web/assets/1389908/f427b473-9456-48df-8a8f-d7e73201d836)

  We don't seem to have useful logs, so I'm not really sure what happened here. In any case it seems likely to be yet another failure mode than the one observed

**Comment by kerryarchibald** [comment]
https://github.com/matrix-org/matrix-js-sdk/actions/runs/6614847990/job/17965700609

**Comment by richvdh** [comment]
https://github.com/matrix-org/matrix-js-sdk/actions/runs/6636711970/job/18029638156#step:8:913

It's very odd that this seems to affect the rust-crypto build so much more than the legacy crypto build. I could believe some sort of systemic "rust is slow" thing going on, but why only this test out of the entire set of read receipts tests?

**Comment by andybalaam** [comment]
Nope, didn't mean to close this. The `skipIfRustCrypto` is still there, and I'm not at all sure this is fixed. Will check.

**Comment by andybalaam** [comment]
I just ran this locally 10 times (with `CYPRESS_RUST_CRYPTO=1`) and it passed every time, so this is a candidate for trying out whether it's fixed.

**Comment by andybalaam** [comment]
Leaving it for now though, because it uses Mark as Read, which we know has problems, that I am hoping to fix next.
